api/models/api.articles.models.js

```diff
@@ -34,17 +35,31 @@ exports.updateArticle = (articleId, body) => {
     });
 };
 
-exports.fetchArticles = (topicsQuery) => {
+exports.fetchArticles = (topic, sortBy = "created_at", order = "DESC") => {
   const queryArr = [];
+  const sortByArray = [
+    "article_id",
+    "author",
+    "title",
+    "body",
+    "topic",
+    "created_at",
+    "votes",
+    "comment_count",
+  ];
+  const orderArray = ["asc", "ASC", "desc", "DESC"];
```

Comment:
What if someone passed in order="Desc" ? Would you want this to work? You can say no it's your API! But you could also convert the order to lowercase and then just have two items in that array.